main/acle.rst

```diff
@@ -5822,3 +5835,183 @@ Instructions
 +---------------------------------+----------------+------------+-----------------+
 
 These intrinsics are available when ``arm_acle.h`` is included.
+
+Architectural Extension Bridges
+###############################
+
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+===============
+
+
+The NEON_SVE Bridge adds intrinsics that allow conversions between NEON and
+SVE vectors.
+
+These intrinsics are defined in the header file ``arm_neon_sve_bridge.h``.
+
+Including this header_file will include ``arm_neon.h`` and ``arm_sve.h``.
+
+Including this header file will define ``__ARM_NEON_SVE_BRIDGE`` as 1.
+
+``svset_neonq``
+---------------
+
+These intrinsics set NEON vector ``subvec`` into the first 128 bit subvector of
```

Comment:
Summarising our off-line discussion:

I agree with @peterwaller-arm that if we're not sure, specifying that the functions copy bit-for-bit (as a .Q) rather than lane-for-lane seems safer.  Users will then have a direct way of getting the behaviour they want, whether that's no code or a shuffle (using an additional Neon intrinsic call).  If instead we define the intrinsics to include a shuffle then users who don't want one would have to do a compensating shuffle and rely on the compiler optimising the sequence away.

If it turns out that the shuffle version is more generally useful, we could add new bridge functions that do that in future.  They would behave the same as the current ones on little-endian.

Reply:
Resolving this and https://github.com/ARM-software/acle/pull/72#discussion_r751218453 as resolved then.

Reply:
Added a section to the design document detailing this decision